core/libp2p/crypto/key_marshaller/key_marshaller_impl.cpp

```diff
@@ -66,37 +66,38 @@ namespace libp2p::crypto::marshaller {
 
   outcome::result<KeyMarshallerImpl::ByteArray> KeyMarshallerImpl::marshal(
       const PublicKey &key) const {
-    proto::PublicKey proto_key;
+    protobuf::PublicKey protobuf_key;
     OUTCOME_TRY(key_type, marshalKeyType(key.type));
-    proto_key.set_key_type(key_type);
-    proto_key.set_key_value(key.data.data(), key.data.size());
+    protobuf_key.set_key_type(key_type);
+    protobuf_key.set_key_value(key.data.data(), key.data.size());
 
-    auto string = proto_key.SerializeAsString();
+    auto string = protobuf_key.SerializeAsString();
     KeyMarshallerImpl::ByteArray out(string.begin(), string.end());
     return KeyMarshallerImpl::ByteArray(string.begin(), string.end());
   }
 
   outcome::result<KeyMarshallerImpl::ByteArray> KeyMarshallerImpl::marshal(
       const PrivateKey &key) const {
-    proto::PublicKey proto_key;
+    // TODO(Harrm): Check if it's a typo
+    protobuf::PublicKey protobuf_key;
```

Comment:
As for me, renaming this variable isn't worth efforts.

Reply:
Actually it was unintended rename due to automatic replace of proto to protobuf, but personally I think that if it's done, it's better to leave it like this, because proto is a common shortcut for protocol which creates unnecessary cognitive load here.

Reply:
Anyway, renaming back would take additional efforts, so it doesn't make sense.